Makefile

```diff
@@ -4,6 +4,10 @@ TAG=$(shell cut -d'=' -f2- .release)
 .DEFAULT_GOAL := build
 .PHONY: release git-tag check-git-status build container-image pre-build tag-image publish test system-check
 
+# Show this help.
+help:
+	@awk '/^#/{c=substr($$0,3);next}c&&/^[[:alpha:]][[:alnum:]_-]+:/{print substr($$1,1,index($$1,":")),c}1{c=0}' $(MAKEFILE_LIST) | column -s: -t
```

Comment:
Please add help as a new target (function) in kind-cluster.sh
and add switchcase  like this to call functions based on commandline args:
```
[ ${#@} -gt 0 ] || help
case "${1}" in
        destory_kind)
            destroy_kind
        ;;
        install_kind)
            install_kind
            ;;
        *)
            help
            exit 1
esac
```

Reply:
This is help added for makefile. It'll help by auto-populating the choices during `make` command.

Reply:
Oh ok, could you please post the output here?

Reply:
```
shabhish@SHABHISH-M-V1CZ botkube (develop)$ make help
help               Show this help.
release            ake a release
git-tag            reate a git tag
check-git-status   heck git status
test               test
build              uild the binary
container-image    uild the image
system-check       ystem checks
pre-build          re-build checks
tag-image          ag images
publish            ocker push image
install-kind       Install KIND
create-kind        Create KIND cluster
destroy-kind       Destroy KIND cluster

```